### PR TITLE
links and copyright date updated

### PR DESCRIPTION
**Pull request by crudmucosa**
Added some links to newer subdomains.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -362,18 +362,52 @@ <h1>Car Shows</h1>
...
+		</div>
+
 		<div class="work">
-			<a href="http://picturesofus.net/">
+			<a href="http://colorado.picturesofus.net/">
 				<img src="img/work1.jpg" class="media" alt=""/>
 				<div class="caption">
 					<div class="work_title">
-						<h1>Place Holder</h1>
+						<h1>Colorado</h1>
```
Comment (smashwilson) [inline index.html:404]: But what about all your stunning photos of Place Holder?!?

**Review by smashwilson — COMMENTED** [review]
Did you mean to close this without merging it? Or did you click the wrong button?

**Comment by crudmucosa** [comment]
Hahaha! Little mix of both. I wanted to see how it worked in the GUI.

**Comment by smashwilson** [comment]
Now check out the review comment 😉

**Pull request closed without merging**